drivers/hmis/app/graphql/types/forms/form_definition.rb

```diff
@@ -33,6 +35,7 @@ class Forms::FormDefinition < Types::BaseObject
     field :date_updated, GraphQL::Types::ISO8601DateTime, null: false, method: :updated_at
     field :date_created, GraphQL::Types::ISO8601DateTime, null: false, method: :created_at
     field :updated_by, Types::Application::User, null: true
+    field :project_matches, [Types::Forms::ProjectMatch], null: false
```

Comment:
There can be hundreds of matching projects, so we should employ pagination https://github.com/greenriver/hmis-frontend/pull/826/files#r1657702742